src/Shared/HttpSys/NativeInterop/HttpApiTypes.cs

```diff
@@ -573,7 +573,7 @@ internal struct HTTP_SERVER_AUTHENTICATION_INFO
             internal bool DisableNTLMCredentialCaching;
             internal ulong ExFlags;
             HTTP_SERVER_AUTHENTICATION_DIGEST_PARAMS DigestParams;
-            HTTP_SERVER_AUTHENTICATION_BASIC_PARAMS BasicParams;
+            readonly HTTP_SERVER_AUTHENTICATION_BASIC_PARAMS BasicParams;
```

Comment:
revert, we should leave the structs alone.

Reply:
I'll clean this up in a follow up.